packages/core/test/oracle/GovernorV2.js

```diff
@@ -165,28 +179,38 @@ describe("GovernorV2", function () {
     // Execute the proposals to clean up.
     const vote = toWei("1");
     const salt = getRandomSignedInt();
-    const hash1 = computeVoteHash({
+    const hash1 = computeVoteHashAncillary({
       price: vote,
       salt,
       account: proposer,
       time: request1.time,
       roundId,
       identifier: request1.identifier,
+      ancillaryData: defaultAncillaryData,
     });
-    const hash2 = computeVoteHash({
+    const hash2 = computeVoteHashAncillary({
       price: vote,
       salt,
       account: proposer,
       time: request2.time,
       roundId,
       identifier: request2.identifier,
+      ancillaryData: defaultAncillaryData,
     });
-    await voting.methods.commitVote(request1.identifier, request1.time, hash1).send({ from: accounts[0] });
-    await voting.methods.commitVote(request2.identifier, request2.time, hash2).send({ from: accounts[0] });
+    await voting.methods
```

Comment:
it's a bit of a pain that these all roll to two lines now but kind of an unavoidable side effect of needing to pass in this extra data.